include/async_mqtt/util/json_like_out.hpp

```diff
@@ -63,7 +63,7 @@ inline std::ostream& operator<<(std::ostream& o, json_like_out_t const& v) {
     return o;
 }
 
-json_like_out_t json_like_out(buffer const& buf) {
+json_like_out_t inline json_like_out(buffer const& buf) {
```

Comment:
```suggestion
inline json_like_out_t json_like_out(buffer const& buf) {
```

Contribution guide is not prepared yet. Coding rules too. But the locaion of the `inline` is the same as other `inline` functions.
Could you update it ?

Reply:
I did